tests/pytests/test_debug_commands.py

```diff
@@ -167,11 +167,13 @@ def testInvertedIndexSummaryWrongArity(self):
     def testNumericIdxIndexSummary(self):
         self.env.expect(debug_cmd(), 'numidx_summary', 'idx', 'age').equal(['numRanges', 1, 'numEntries', 1,
                                                                            'lastDocId', 1, 'revisionId', 0,
-                                                                           'emptyLeaves', 0, 'RootMaxDepth', 0])
+                                                                           'emptyLeaves', 0, 'RootMaxDepth', 0,
+                                                                           'MemoryUsage', ANY])
```

Comment:
why ANY?

Reply:
Just to not bother with an arbitrary byte count. I think getting the reply structure is more important.
But I can add it in, now that the implementation is complete and the memory consumption should be constant